docs/scos/dev/front-end-development/202212.0/oryx/reactivity/reactive-components.md

```diff
@@ -45,30 +45,25 @@ While the user navigates thorugh a single page application, it is crucial for co
+The components provided in the Oryx libraries are built with Lit. Lit provides a highly efficient system to only synchronize the minimum required updates to the DOM. When updates are loaded asynchronously, the UI needs to be updated every time new data is emitted. To make this as transparent as possible, the `@signalAware()` decorator is used in the UI components, making possible to use `signals` when building UI, which automatically requests updates to the view when needed.
 
-The following example shows the usage of the `asyncState` decorator. The decorator subscribes to the assigned observable and requests an update to the component when needed. This means that component developers do not need to worry about how the reactive system works under the hood.
-
-Oryx components are built in TypeScript, and we provide types everywhere to increase the developer experience and avoid errors upfront. The original type of the assigned observable needs to be adjusted. It's impossible to resolve a correct type from the observable by using a decorator, which is why the `valueType` function is used to resolve the observed type.
+The following example shows the usage of the `computed` signal. `Computed` wraps observable into `signal` that will subscribe to the underlying observable automatically, and trigger updating the view.  This means that component developers do not need to worry about how the reactive system works under the hood.
 
 ```ts
 export class ProductPriceComponent {
-  @asyncState()
-  protected prices = valueType(
-    this.productController
-      .getProduct()
-      .pipe(switchMap((product) => this.formatPrices(product?.price)))
-  );
+    protected $prices = computed(() => {
+        return this.formatPrices(this.$product()?.price);
+    });
 
   protected override render(): TemplateResult | void {
-    return html`${this.prices.defaultPrice}`;
+    return html`${this.$prices().defaultPrice}`;
   }
 }
 ```
 
 ## Multiple data streams
 
-Components often use multiple data streams. For example, the product price component renders a product price in a certain currency and a _local_ price format. The currency and locale are part of the application context and may change during the application's lifecycle. The product price changes from product to product. RxJS operators combine the various streams. They can combine multiple observables and operate on the combined results.
+Components often use multiple data streams. For example, the product price component renders a product price in a certain currency and a _local_ price format. The currency and locale are part of the application context and may change during the application's lifecycle. The product price changes from product to product. You can use both signals and RxJS decorator to combine the various streams. They can combine multiple observables and operate on the combined results.
```

Comment:
RxJS decorator is not a thing, moreover I'd either skip it, or add it in a side note.

Reply:
👍